include/Menu.h

```diff
@@ -11,7 +12,7 @@ class Menu : public State
   public:
     //Menu();
 
-    Menu(sf::RenderWindow& app);
+    Menu(sf::RenderWindow& app, Sound *sound);
```

Comment:
we shouldn't need to pass a sound into the constructor

Reply:
I recommend making a setSound method inside State.h and having GameView, Menu, all the other states inherit from it.